src/kernel/kmain.c

```diff
@@ -72,10 +76,10 @@ void check_interrupts()
 
 void kmain(uint64_t addr)
 {
-  console_init();
-
   multiboot_info_t* mbi = (multiboot_info_t*)addr;
 
+  console_init();
```

Comment:
Could we use the `vga_text` driver until we load the vesa driver?

Reply:
Unfortunately, grub will setup the framebuffer for us immediately so we can't.